lib/compiler.js

```diff
@@ -15,8 +15,14 @@ Compiler.prototype.generate = function generate(code, options) {
     code = '';
 
   // Yeah, let people pass functions to us!
-  if (typeof code === 'function')
-    code = code.toString().replace(/^function\s*\w*\(\)\s*{|}$/g, '');
+  if (typeof code === 'function') {
+    code = code.toString();
+    code = code.replace(code.indexOf('function') === 0 ?
+      // Regular function
+      /^function\s*\w*\(\)\s*{|}$/g :
+      // Arrow function
+      /^\(\)\s=>\s{|}$/g, '');
```

Comment:
Please add support for `_ => {}`.

Hm, we don't have variables at all in the `function(){ … }` variant.

Btw, there could be spaces between parens. We could switch to `function\s*[^{]+{…` and `(\w|[^=>]+)\s*=>\s*\{` — these ones are relaxed variants for the first one, they could be much more useful for demo.